src/main/java/cmu/pasta/mu2/fuzz/MutationGuidance.java

```diff
@@ -163,6 +169,7 @@ public void run(TestClass testClass, FrameworkMethod method, Object[] args) thro
     byte[] argBytes = Serializer.serialize(args);
     int run = 1;
 
+    runMutants = filterMutants();
```

Comment:
Similar to my comment for #55, we should either have a purely functional application where a set is passed in and a new one is returned, or if `filterMutants` is working on the `this.runMutants` data structure then the method should not return anything.